site/zenodo_rdm/migrator/extract.py

```diff
@@ -231,60 +231,6 @@ def run(self):
 ) TO STDOUT;
 """
 
-# TODO: Check if we just dump the entire tables (files, objects, buckets) and
```

Comment:
Removing it now, will be re-added (slightly modified) as a post_load hook for draft files